### ISSUE TITLE
Panic when using mun build

### ISSUE THREAD

**Issue by sigmaSd** [mun-lang/mun#228]
```rust
pub  fn fact(n: usize) -> usize {
   	if n == 0 {return 1} else {n * x(n-1)}
}
```

This panics here https://github.com/TheDan64/inkwell/blob/master/src/module.rs#L1383

Removing the `return` keyword  or adding `return` in the else clause fixes the issue.

**Comment by baszalmstra** [comment]
Thanks for the report! It actually highlighted a couple of bugs! I fixed them in #229

**Comment by sigmaSd** [comment]
Glad it helped!
